pkg/tsdbctl/create.go

```diff
@@ -62,7 +62,12 @@ func newCreateCommandeer(rootCommandeer *RootCommandeer) *createCommandeer {
 
 	cmd := &cobra.Command{
 		Use:   "create",
-		Short: "create a new TSDB in the specifies path",
+		Short: "Create a new TSDB instance",
+		Long:  `Create a new TSDB instance (table) according to the provided configuration.`,
+		Example: `- tsdbctl create -s 192.168.1.100:8081 -u myuser -p mypassword -c mycontainer -t my_tsdb --rate 1/s
```

Comment:
Change `--rate` to `--ingestion-rate`

Reply:
I saw the change, thanks. The help text for this flag in my PR is actually longer than what you used (that's why I wrote "..." at the end of the quoted text in my comment), but I'll handle this when merging the PR #141 changes from `development` to my PR branch.